packages/runtime/src/state/makeswift-api-client.ts

```diff
@@ -129,5 +132,20 @@ export type Dispatch = ThunkDispatch<State, unknown, Action>
 export type Store = ReduxStore<State, Action> & { dispatch: Dispatch }
 
 export function configureStore({ serializedState }: { serializedState?: SerializedState }): Store {
-  return createStore(reducer, APIResources.getInitialState(serializedState), applyMiddleware(thunk))
+  const composeEnhancers = composeWithDevToolsDevelopmentOnly({
+    name: `API client store (${new Date().toISOString()})`,
+    serialize: true,
+    stateSanitizer: (state: any) => serializeState(state),
```

Comment:
Even with `serialize: true`, Redux Devtools doesn't fully show state slices that use `Map` or `Set` to manage the state; use `stateSanitizer` to serialize the state into a plain JSON tree.